packages/im-ott/src/organisms/filmList/FilmList.js

```diff
@@ -126,9 +126,7 @@ const FilmList = (props) => {
           {label} &nbsp; <RightOutlined />
         </div>
         <div className={styles.seeMore} onClick={handleSeeAll}>
-          <p>
-            See all <RightOutlined />
-          </p>
+          <p>See all</p>
```

Comment:
why remove > ? 
it was looking good right

Reply:
Removed only the arrow key. Was not looking good with See all